web/packages/teleport/src/Discover/Database/MutualTls/useMutualTls.ts

```diff
@@ -113,7 +113,8 @@ function generateSignCertificateCurlCommand(
   if (!token) return '';
 
   const requestUrl = cfg.getDatabaseSignUrl(clusterId);
-  const requestData = JSON.stringify({ hostname });
+  const ttl = cfg.getDatabaseCertificateTTL();
+  const requestData = JSON.stringify({ hostname, ttl });
```

Comment:
at the top of this file, i would define

```ts
// the length of the certificate to request for the database
const DEFAULT_DATABASE_TTL = '2190h'
```

then

```suggestion
  const requestData = JSON.stringify({ hostname, ttl: DEFAULT_DATABASE_TTL });
```